ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/AwsSecurityHubIntegrationForm.tsx

```diff
@@ -15,6 +15,7 @@ import { IntegrationFormProps } from '../integrationFormTypes';
 import IntegrationFormActions from '../IntegrationFormActions';
 import FormLabelGroup from '../FormLabelGroup';
 import AwsRegionOptions from '../AwsRegionOptions';
+import useFeatureFlags from '../../../../hooks/useFeatureFlags';
```

Comment:
Because this is import from outside of Integrations, can you move following line 11 and rewrite:

```ts
import useFeatureFlags from 'hooks/useFeatureFlags';
```